homeassistant/components/openai_conversation/conversation.py

```diff
@@ -138,6 +139,10 @@ async def _transform_stream(
                     )
                 ]
             }
+        elif isinstance(event, ResponseIncompleteEvent):
+            raise HomeAssistantError(
+                f"OpenAI response incomplete: {event.response.incomplete_details.reason if event.response.incomplete_details else None}"
+            )
```

Comment:
For a future PR: Should we do an `else` and warning log unhandled event?

Reply:
But you are right that there are more events that we should probably handle.

Reply:
I've added a few more error events for completeness